packages/langium-cli/src/langium.ts

```diff
@@ -4,20 +4,36 @@
  * terms of the MIT License, which is available in the project root.
  ******************************************************************************/
 
+import 'colors';
 import { Command } from 'commander';
+import { validate } from 'jsonschema';
 import { generate, GenerateOptions } from './generate';
+import { cliVersion, elapsedTime, schema } from './generator/util';
+import { LangiumConfig, loadConfigs } from './package';
 
 const program = new Command();
 program
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    .version(require('../package.json').version);
-
-program
+    .version(cliVersion)
     .command('generate')
-    .description('generate code for a Langium grammar')
+    .description('Generate code for a Langium grammar')
     .option('-f, --file <file>', 'the configuration file or package.json setting up the generator')
     .action((options: GenerateOptions) => {
-        generate(options);
+        forEachConfig(options, generate);
+        console.log(`Langium generator finished ${'successfully'.green.bold} in: ${elapsedTime()}ms`);
     });
 
 program.parse(process.argv);
+
+function forEachConfig(options: GenerateOptions, callback: (config: LangiumConfig) => void): void {
+    const configs = loadConfigs(options.file);
+    if (!configs.length) {
+        console.error('Could not find a langium configuration. Please add a langium-config.json to your project or a langium section to your package.json.'.red);
+        process.exit(1);
+    }
+    const validation = validate(configs, schema);
+    if (!validation.valid) {
+        console.error('Your langium configuration is invalid.'.red);
+        process.exit(1);
+    }
+    configs.forEach(callback);
```

Comment:
I think we will need to change this when we support multiple languages, but it's ok to process each language independently for now.